snap/info_snap_yaml.go

```diff
@@ -218,12 +223,13 @@ func setSlotsFromSnapYaml(y snapYaml, snap *Info) error {
 	return nil
 }
 
-func setAppsFromSnapYaml(y snapYaml, snap *Info) {
+func setAppsAndAliasesFromSnapYaml(y snapYaml, snap *Info) error {
```

Comment:
New function name feels oververbose. Keeping the old one but including the new logic would seem alright.